Centaurus.Test.Utils/SnapshotHelper.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Centaurus.Domain;
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Centaurus.Test
+{
+    public static class SnapshotHelper
+    {
+        public static async Task ApplyUpdates()
+        {
+            var globalType = typeof(Global);
```

Comment:
Move these two lines into the static constructor and save the reference to `applyUpdatesMethod` into the private static variable. **DO NOT** use uncached reflection discovery in any code that can be potentially executed more than once, even if it doesn't create a performance bottleneck right now.